doc/reference/bluetooth/mesh/shell.rst

```diff
@@ -22,9 +22,9 @@ See :ref:`shell_api` for information on how to connect and interact with the Blu
 Basic usage
 ***********
 
-The Bluetooth mesh shell subsystem adds a single ``mesh`` command, which holds a set of sub-commands. Every time the device boots up, make sure to call ``mesh init`` before any of the other Bluetooth mesh shell commands can be called::
+The Bluetooth mesh shell subsystem adds a single ``mesh`` command, which holds a set of sub-commands. Every time the device boots up, make sure to call ``mesh init-shell`` before any of the other Bluetooth mesh shell commands can be called::
 
-	uart:~$ mesh init
+	uart:~$ mesh init-shell
```

Comment:
IMO, it's better not to change the name here. I get the motivation, but this would be the only change that breaks the current workflow. As long as the point of this is sufficiently documented, the old name isn't *wrong*